lib/tasks/statistics.rake

```diff
@@ -3,16 +3,56 @@ namespace :statistics do
   task :general, %i[start_date end_date] => [:environment] do |_, args|
     args.with_defaults(start_date: Date.yesterday.midnight.to_s, end_date: Date.yesterday.end_of_day.to_s)
 
-    total_users = User
+    all_users = User
       .where("created_at < ?", args.end_date)
+
+    total_users = all_users
       .count
     puts "All registrations to #{args.end_date}: \n#{total_users}\n\n"
 
-    new_users = User
+    all_new_users = User
       .where("created_at BETWEEN ? AND ?", args.start_date, args.end_date)
+
+    new_users = all_new_users
       .count
     puts "New registrations between #{args.start_date} and #{args.end_date}: \n#{new_users}\n\n"
 
+    all_cookie_consent = all_users
+      .pluck(:cookie_consent)
+      .tally
```

Comment:
:D I've been wanting to give this a go. This was new in 2.6 or 2.7 wasn't it?

Reply:
I think it was 2.7.  I've used it so many times already, I've no idea why it wasn't part of older versions!